job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/step/PhaseJobContext.groovy

```diff
@@ -123,6 +124,14 @@ class PhaseJobContext implements Context {
      * @since 1.25
      */
     @RequiresPlugin(id = 'jenkins-multijob-plugin', minimumVersion = '1.11')
+    void abortAllJob(boolean abortAllJob = true) {
```

Comment:
The flag has been introduced in 1.14, see https://github.com/jenkinsci/tikal-multijob-plugin/commit/4260fc5c317ea1328129777e0987a2fceceffd15#diff-5a6aed1c369f1f6d85ff54874b8e2e8f. Please fix the minimumVersion and also update the since tag.